src/app/api/__tests__/mocks/prisma.ts

```
@@ -0,0 +1,21 @@
+import { PrismaClient } from '@prisma/client';
+
+type MockPrismaFunction = jest.Mock<any, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)